solr/core/src/java/org/apache/solr/update/DirectUpdateHandler2.java

```diff
@@ -262,27 +262,29 @@ public void initializeMetrics(
             baseCommandsMetric,
             baseAttributes.get().put(OPERATION_ATTR, "deletes_by_query").build());
 
+    var baseCommitMetric =
```

Comment:
Why is this a "base"; why do you refer to it in the metrics for commands & merges below other than "commits"?